create-container.sh

```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
   BASE_PATH="/opt"
 fi
 
+# Test if known_hosts file exists
+if [ ! -f ~/.ssh/know_hosts ] ; then
+  touch ~/.ssh/know_hosts
+fi
+
+# Test if public key created
+if [ ! -f ~/.ssh/id_rsa.pub ] ; then
+  ssh-keygen -t rsa -N "" -q -f "$HOME/.ssh/id_rsa"
```

Comment:
```suggestion
  echo "I can't find a SSH public key in the default path: `$HOME/.ssh/id_rsa`."
  echo "You can use the var SSH_KEY_PATH to set a different path if you don't use the default."
  exit 0
```

Reply:
I liked this way!

Reply:
@jsarola this change also disappered, right?